specs/Workers.md

```diff
@@ -0,0 +1,1402 @@
+      [in] ICoreWebView2DedicatedWorkerCreatedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_DedicatedWorkerCreated`.
+  HRESULT remove_DedicatedWorkerCreated(
+      [in] EventRegistrationToken token);
+}
+
+/// Receives `DedicatedWorkerCreated` events.
+[uuid(99c76d22-d2de-5b04-b8e5-07b27584da49), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorkerCreatedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2* sender,
+      [in] ICoreWebView2DedicatedWorkerCreatedEventArgs* args);
+}
+
+/// Event args for the `DedicatedWorkerCreated` event.
+[uuid(4a122222-390e-5a65-809c-f043486db602), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorkerCreatedEventArgs : IUnknown {
+  /// The dedicated worker that was created.
+  [propget] HRESULT Worker([out, retval] ICoreWebView2DedicatedWorker** value);
+}
+
+/// Receives `Destroyed` events.
+[uuid(d3066b89-9039-5ffc-b594-f936773d5bc5), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorkerDestroyedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2DedicatedWorker* sender,
+      [in] IUnknown* args);
+}
+
+[uuid(f233edb4-d9b4-5209-9abd-b7f50089464c), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorker : IUnknown {
+  /// A string representing the Uri of the script that the worker is executing.
+  ///
+  /// This corresponds to the `scriptURL` property of the `Worker` object in the DOM.
+  /// The `scriptURL` property returns a string representing the URL of the script that the
+  /// worker is executing. See the [MDN documentation]
+  /// (https://developer.mozilla.org/docs/Web/API/Worker/scriptURL) for more information.
+  ///
+  /// The caller must free the returned string with `CoTaskMemFree`.  See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings).
+  [propget] HRESULT ScriptUri([out, retval] LPWSTR* value);
+
+  /// Add an event handler for the `Destroyed` event that is raised when the
+  /// worker object is destroyed.
+  ///
+  /// A worker object is destroyed when the worker script is terminated or when
+  /// the `CoreWebView2DedicatedWorker` object is destroyed.
+  HRESULT add_Destroyed(
+      [in] ICoreWebView2DedicatedWorkerDestroyedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_Destroyed`.
+  HRESULT remove_Destroyed(
+      [in] EventRegistrationToken token);
+}
+
+
+[uuid(ad6921d4-c416-5945-8437-2c97aeb76e6e), object, pointer_default(unique)]
+interface ICoreWebView2Profile2 : IUnknown {
+  /// Get the service worker manager to monitor service worker registrations and
+  /// interact with the service worker associated with the current profile.
```

Comment:
```suggestion
  /// interact with the service workers associated with the current profile.
```
Plural I assume. You can have multiple service workers associated with a profile.